test/e2e/e2e_suite_test.go

```diff
@@ -36,22 +29,42 @@ var (
 	hubKubeClient      kubernetes.Interface
 	hubAddOnClient     addonclient.Interface
 	hubClusterClient   clusterclient.Interface
-	clusterCfg         *rest.Config
+
+	hostedKlusterletName              string // name of the hosted mode klusterlet
```

Comment:
can we move this variable in the agent_hosting_deploy_test file? The initialization of these files should also only be put in the that files

Reply:
but these are some preparation work, only need to be done once, if we move them to the ginkgo.Describe, they will be called multiple times.

Reply:
moved all hosted e2e to another folder.